js/src/tooltip.js

```diff
@@ -472,8 +470,7 @@ class Tooltip extends BaseComponent {
         {
           name: 'flip',
           options: {
-            altBoundary: true,
-            fallbackPlacements: this.config.fallbackPlacements
+            altBoundary: true
```

Comment:
You removed the `fallbackPlacemens` option here :thinking:

Reply:
You are right. Excuse me, I wrote it on the issue and forgot to transfer it here

Both  solutions, exclude many fallback tooltip placements on popper .
If the option is omitted, it allows  the tooltip, to flip only in the opposite of its initial side side. (which is not so bad)**

Reply:
Ref: https://github.com/twbs/bootstrap/pull/32843#issue-557574003

Reply:
This change is unrelated IMO :thinking: 
/CC @XhmikosR

Reply:
I reverted it for now and rebased the branch. @GeoSot feel free to bring it back later if needed.